### DISCUSSION TITLE
[Locked] macOS: packages installer tool

### DISCUSSION THREAD

**Discussion by JanosGit** [actions/virtual-environments#2460] [Installed software]
I'm using the [packages](http://s.sudre.free.fr/Software/Packages/about.html) ([github.com/packagesdev/packages](https://github.com/packagesdev/packages)) installer builder to build macOS installers for my projects. At least in the audio software community where I'm coming from, this tool is used quite commonly to build macOS installers. As it's not available via Homebrew, runtime installation hast to be done manually in my workflows (downloading the file extracting the image, installing it with root rights, which feels a bit "wrong" ;))

Is there any chance this software could be added to the installed software for the macOS runners or is it just not used widespread enough in order to justify that?

**Comment by AlenaSviridenko** [comment]
Hi @JanosGit,
thanks for provided explanation of where the tool is used, but we'd rather not to add this to the image due to maintenance concerns and tool popularity. We are trying to support complex installation without package managers only for big tools. 
If the tool can be installed in runtime with less than a minute, we are suggesting installing it that way or use action if it exists.
Thanks!
